libr/core/anal_vt.c

```diff
@@ -36,8 +39,10 @@ static int inTextSection(RCore *core, ut64 curAddress) {
 }
 
 static int valueInTextSection(RCore *core, ut64 curAddress) {
+	int sz = R_DIM (8, 1, 8);
 	//value at the current address
-	ut64 curAddressValue = r_io_read_i (core->io, curAddress, 8);
+	ut64 curAddressValue;
+	r_io_read_at (core->io, curAddress, (ut8 *)&curAddressValue, sz);
```

Comment:
this thing have a clear endianness problem, so we need a way to read that using r_read_le64 and such